frontend/src/data/variables/AcsPopulationProvider.ts

```diff
@@ -144,15 +173,19 @@ class AcsPopulationProvider extends VariableProvider {
   }
 
   allowsBreakdowns(breakdowns: Breakdowns): boolean {
-    const validDemographicBreakdownRequest: boolean =
+    const validGeographicBreakdown =
+      breakdowns.geography === "county"
+        ? breakdowns.demographicBreakdowns.race_nonstandard.enabled ||
+          breakdowns.demographicBreakdowns.race.enabled
```

Comment:
Use helper methods `hasOnlyRaceNonStandard` and `hasOnlyRace`?

Reply:
oh ya! little merge artifact...